src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -59,26 +59,26 @@ public class Drivetrain extends SubsystemBase {
     public static final double MAX_ENCODER_VELOCITY = 20743.0d;
     public static final double KF_TYPICAL_PERCENT_USAGE = 0.75d; // We will typically use 75% of max speed
     public static final double TALON_FULL_OUTPUT_SETTING = 1023;
-    public static final double VELOCITY_KF = 0.046d;
-    public static final double VELOCITY_KP = 0.03d;
+    public static final double VELOCITY_KF = 0.0d;
```

Comment:
We tested the PID today and found these numbers are very close. Let's leave them as is for now.

Reply:
Fixed